src/renderer/controllers/torrent-list-controller.js

```diff
@@ -40,6 +40,11 @@ module.exports = class TorrentListController {
 
   // Shows the Create Torrent page with options to seed a given file or folder
   showCreateTorrent (files) {
+    // You can only create torrents from the home screen.
+    if (this.state.location.url() !== 'home') {
+      return dispatch('error', 'Please go back to the torrent list before creating a new torrent.')
```

Comment:
Oh, that's bad. Okay, this is fine then.

Btw right now, you also can't open Prefs from within the Create Torrent page.